odoo/service/db.py

```diff
@@ -383,12 +383,26 @@ def list_dbs(force=False):
         res = sorted(db.strip() for db in odoo.tools.config['db_name'].split(','))
         return res
 
-    chosen_template = odoo.tools.config['db_template']
-    templates_list = tuple(set(['postgres', chosen_template]))
-    db = odoo.sql_db.db_connect('postgres')
+    unlisted_dbs = {'postgres', odoo.tools.config['db_template']}
+    if odoo.tools.config['db_name'] != odoo.tools.config['db_meta']:
+        unlisted_dbs.add(odoo.tools.config['db_meta'])
```

Comment:
It doesn't look correct to me:
Lets imageine that the db_meta is one of the real database, and is part of the db_filter

My command line is `--db_meta db_1n --db_filter db.*`
 
 -> db_1 won't appear in the database manager if I'm correct

Reply:
indeed it was not correct, the meta-db is now listed but it is unaccessible. Selecting it or removing it does nothing. A bit of noise in the db-list is the best I can do as I cannot use `http.db_filter` in this context due to the lack of hostname to set (the request may be missing)